components/brave_wallet_ui/components/desktop/popup-modals/account-settings-modal/account-settings-modal.tsx

```diff
@@ -274,52 +275,70 @@ export const AccountSettingsModal = () => {
           <DepositModal selectedAccount={selectedAccount} />
         )}
         {accountModalType === 'edit' && (
-          <>
+          <EditWrapper>
             <Input
               value={accountName}
               placeholder={getLocale('braveWalletAddAccountPlaceholder')}
-              onChange={handleAccountNameChanged}
+              onInput={handleAccountNameChanged}
               onKeyDown={handleKeyDown}
-            />
+              showErrors={showNameInputErrors}
+              maxlength={30}
+              size='large'
+            >
+              {getLocale('braveWalletAddAccountPlaceholder')}
+              <div slot='errors'>
+                <ErrorText>
+                  {isAccountNameTooLong
+                    ? getLocale('braveWalletAccountNameTooLongError')
+                    : ''}
+                </ErrorText>
+              </div>
+              <div slot='extra'>
+                {isAccountNameTooLong ? (
+                  <ErrorText>{accountName.length}/30</ErrorText>
```

Comment:
Let's make it a global constant. 
brave_wallet.mojom might be a good place to make it reusable by other ui platforms

Reply:
fixed: [ee8dc61](https://github.com/brave/brave-core/pull/22907/commits/ee8dc614db67d0e4ca93d89d394288e437c7adac)